### ISSUE TITLE
Refine narrative?

### ISSUE THREAD

**Issue by jeromekelleher** [petrelharp/num_edges#19]
As I mentioned in a note in the abstract in #18 and (#16, #17) , I think we're not quite hitting the right note narrative wise. In the abstract we're emphasising the application to inference, but that's really the one thing that we're *not* doing effectively. The application to tsinfer basically just shows (I think?) that tsinfer is doing OK in this respect and that using extend_haplotypes doesn't obviously make things better.

The question about whether this would make Relate or even SINGER better is really hanging, so we have to make this part of the narrative unless we want to be assigned this for revisions (if we're lucky).

I think the way to do this is to frame the paper as setting the basic theoretical basis for addressing these questions, showing that that unary nodes can be imputed when we have perfect data from simulations. We can make a lot more of the fact that simularity/dissimilarity measures are developed (which reads very much like an aside at the moment), and the tscompare package (which exists??) provides the practical basis on which to evaluate the ARGs produced by different methods in a more holistic way (i.e., less of the narrow focus on tree-by-tree measures).

I don't think this would take much doing, it's just a bit of editing on the key paragraphs in the abstract, introduction and discussion.

**Comment by petrelharp** [comment]
I like it!

**Comment by petrelharp** [comment]
Probably remove "inferred" from the title.

**Comment by petrelharp** [comment]
> % JK: The only application to inference we have is from tsinfer, where the results are pretty equivocal.

That comment is in response to this, from the abstract: 

>     In this paper, we describe how this recombination-mediated information
>    can in many cases be recovered from inference based solely on polymorphic markers.

But, in this sentence "inference" refers to the extend_haplotypes method itself. Like, as we extend haplotypes, we're inferring the existence of these unary spans.  Perhaps we should say "can in many cases be recovered by an algorithm that uses only polymorphic markers"?

**Comment by jeromekelleher** [comment]
Right, I guess this is the root of my confusion. I think "inference" is a bit too strong here and "imputation" would be a better word. So, given an ARG in (usually) fully simplified form we can impute the true span of ancestral haplotypes based on a parsimony argument using this algorithm. 

Talking about polymorphic markers here implies to me that we're actually looking at the raw variation data, which we're not.

**Comment by petrelharp** [comment]
@nspope suggests changing the title to "A forest is more than... : haplotypes and ancestral recombination graphs"

**Comment by petrelharp** [comment]
And, I've had a fairly close look at this one. I totally agree that that sentence in the abstract was confusing. Over in #30 I've essentially removed it. I also had a good think about rewriting the abstract, and didn't come up with anything better. As written, it's trying to frame things just like you're suggesting, i.e., by "showing that that unary nodes can be imputed when we have perfect data." We could turn the abstract the other way around, first saying "ARG work usually just focuses on trees, which ignores haplotypes, so we define some metrics; also we can infer haplotypes" - but that's not how the paper's written, either?

So, I'm inclined to say what I did in #30 addresses this - but, more suggestions welcome?

**Comment by jeromekelleher** [comment]
Yep, looks good :+1:
